.env

```diff
@@ -0,0 +1,6 @@
+MONGODB_DIR=/usr/local/var/lib/docker/mongodb
+HOTSTORAGE_USERNAME=<mongodb-user>
+HOTSTORAGE_PASSWORD=<mongodb-password>
+HOTSTORAGE_NAME=<mongodb-database>
+IS_VERSION=5.3.0
```

Comment:
I think we can remove `IS_VERSION` here. Are we using hotstorage env variables to this deployment?

Reply:
Good point, I'll remove them